mentors/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
+
+RUN apt update && \
```

Comment:
you only need one `install` here. As a style rule, we order packages in alpha asc, and then at the end of an install do the clean up, so the whole RUN should look like this:

```
RUN apt-get update && \
    apt-get install -y \
        ffmpeg \
        python3.7 \
        python3-pip && \
    rm -rf /var/lib/apt/lists/*
```